F21/mukthipola345/mukthi.py.py

```diff
@@ -0,0 +1,24 @@
+l=input()
+word=input()
+res=[]
+
+if l=="en" or l=="en-US" or l=="en-Latn" or l=="en-IE":
+    res=word.lower()
+
+elif l=="ga" or l=="ga-IE":
+    if word[0] in ['n','t'] and word[1] in ['A','E','I','O','U']:
+        res.append(word[0])
+        res.append('-')
+        for i in word[1:]:
+            res.append(i.lower())
+    else:
+        res=word.lower()
+elif l=="el":
```

Comment:
you have just five symbols got executed in this language ensure different symbols likewise covered in text file  should be executed.